autosubmit/notifications/mail_notifier.py

```diff
@@ -20,18 +20,30 @@
 import smtplib
 import email.utils
 from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from email.mime.application import MIMEApplication
 from log.log import Log
+from pathlib import Path
+from autosubmitconfigparser.config.basicconfig import BasicConfig
 
 class MailNotifier:
     def __init__(self, basic_config):
         self.config = basic_config
 
     def notify_experiment_status(self, exp_id,mail_to,platform):
         message_text = self._generate_message_experiment_status(exp_id, platform)
-        message = MIMEText(message_text)
+        message = MIMEMultipart() 
```

Comment:
We need to confirm if this is the best option. I remember the discussion, but I didn't have time to ponder about it, sorry.

@mcastril, here we have a few options, basically,

1) we send a notification as we already do, but include the location of the logs in the VM/hub/somewhere
2) we attach the complete log as-is into the email body
3) we compress and then attach the logs (and delete the compressed archive as the original logs will remain anyway) into the email body

I think I am more included towards 1) to simplify or a 1.5) option, where we let users choose whether they want the location or the attachment, but I had seen a comment about doing the option 2), which is implemented here.

WDYT, Miguel?